Dockerfile

```diff
@@ -53,11 +53,15 @@ RUN mkdir -p /tmp/run_artifacts
 # download benchmark-operator to /tmp default path
 RUN git clone https://github.com/cloud-bulldozer/benchmark-operator /tmp/benchmark-operator
 
+# download cluster-buster to /tmp default path && install cluster-buster dependency
+RUN git clone https://github.com/RobertKrawitz/OpenShift4-tools /tmp/OpenShift4-tools \
+    && dnf install -y hostname bc
+
 # Add main
 COPY benchmark_runner/main/main.py /benchmark_runner/main/main.py
 
 CMD [ "python3.9", "/benchmark_runner/main/main.py"]
 
 # oc: https://www.ibm.com/docs/en/fci/6.5.1?topic=steps-setting-up-installation-server
 # sudo podman build -t quay.io/ebattat/benchmark-runner:latest . --no-cache
-# sudo podman run --rm -it -v /root/.kube/:/root/.kube/ -v /etc/hosts:/etc/hosts --privileged quay.io/ebattat/benchmark-runner:latest /bin/bash
+# sudo podman run --rm -it -v /root/.kube/:/root/.kube/ -v /etc/hosts:/etc/hosts --privileged quay.io/ebattat/benchmark-runner:latest /bin/bash
```

Comment:
Please configure your IDE to add a newline to the last line of a file.  That's the Linux standard.